docs/en/platform-engineers/cloud-services.md

```diff
@@ -58,86 +65,121 @@ EOF
 
 ## Step 3: Verify
 
-Check if the new workload type is added:
-
-```console
-$ vela workloads
-Synchronizing capabilities from cluster⌛ ...
-Sync capabilities successfully ✅ Add(1) Update(0) Delete(0)
-TYPE	CATEGORY	DESCRIPTION     
-+rds	workload	RDS on Ali Cloud
+Use RDS component in an [Application](../application.md) to provide cloud resources.
 
-Listing workload capabilities ...
+As we have claimed an RDS instance with ComponentDefinition name `rds`.
+The component in the application should refer to this type.
 
-NAME      	DESCRIPTION                             
-rds       	RDS on Ali Cloud                        
-task      	One-time task/job                       
-webservice	Long running service with network routes
-worker    	Backend worker without ports exposed    
+```yaml
+apiVersion: core.oam.dev/v1beta1
+kind: Application
+metadata:
+  name: mydatabase
+spec:
+  components:
+    - name: myrds
+      type: rds
+      properties:
+        name: "alibaba-rds"
+        storage: 20
+        secretname: "myrds-conn"
```

Comment:
Change `outputSecretName` to `secretName`?

Reply:
no need to keep `outputSecretName` mechanism for now